core/plugins/org.fusesource.ide.foundation.ui/src/org/fusesource/ide/foundation/ui/io/CamelContextNodeEditorInput.java

```diff
@@ -89,4 +89,5 @@ public void pushbackToRemoteContext(String xml) {
 	private boolean isCamelContextNode(Node contextNode) {
 		return "org.fusesource.ide.jmx.camel.navigator.CamelContextNode".equals(contextNode.getClass().getName());
```

Comment:
I made even better in my other PR, I moved the class and this equal is not even needed so i removed it